File: homepage/src/main/java/com/example/homepage/recyclerview/item/MySection.java
package com.example.homepage.recyclerview.item;

import com.chad.library.adapter.base.entity.SectionEntity;

/**
 * created by tea9 at 2018/11/16
 */
public class MySection extends SectionEntity<Video> {

    private boolean isMore;
    public MySection(boolean isHeader, String header) {
        super(isHeader, header);
    }

    public MySection(Video t) {
        super(t);
    }


}

File: homepage/src/main/java/com/example/homepage/entity/MySection.kt
package com.example.homepage.entity

import com.chad.library.adapter.base.entity.SectionEntity

/**
 * created by tea9 at 2018/9/18
 */
class MySection : SectionEntity<String> {
    var isMore: Boolean = false

    constructor(isHeader: Boolean, header: String, isMroe: Boolean) : super(isHeader, header) {
        this.isMore = isMroe
    }

    constructor(t: String) : super(t) {}
}